manager/assets/modext/widgets/resource/modx.panel.trash.js

```diff
@@ -0,0 +1,359 @@
+MODx.panel.Trash = function (config) {
+    config = config || {};
+    Ext.applyIf(config, {
+        id: 'modx-panel-trash'
+        , cls: 'container'
+        , bodyStyle: ''
+        , defaults: {collapsible: false, autoHeight: true}
+        , items: [{
+            html: _('trash.page_title')
+            , id: 'modx-trash-header'
+            , xtype: 'modx-header'
+        }, MODx.getPageStructure([{
+            layout: 'form'
+            , title: _('trash.tab_title')
+            , items: [{
+                html: '<p>' + _('trash.intro_msg') + '</p>'
+                , xtype: 'modx-description'
+            }, {
+                xtype: 'modx-grid-trash'
+                , id: 'modx-trash-resourcelist'
+                , cls: 'main-wrapper'
+                , preventRender: true
+            }]
+        }], {
+            stateful: true
+            , stateId: 'modx-trash-tabpanel'
+            , stateEvents: ['tabchange']
+            , getState: function () {
+                return {activeTab: this.items.indexOf(this.getActiveTab())};
+            }
+        })]
```

Comment:
If you do not want the tab panel, just use your form panel : replace the whole `MODx.getPageStructure(` with

```
{
            layout: 'form'
            , title: _('trash.tab_title')
            , items: [{
                html: '<p>' + _('trash.intro_msg') + '</p>'
                , xtype: 'modx-description'
            }, {
                xtype: 'modx-grid-trash'
                , id: 'modx-trash-resourcelist'
                , cls: 'main-wrapper'
                , preventRender: true
            }]
        }
```